lib/models/asset-size-printer.js

```diff
@@ -63,17 +63,13 @@ module.exports = class AssetPrinterSize {
         })
         // Print human-readable file sizes (including gzipped)
         .map((file) => {
-          let contentsBuffer = fs.readFileSync(file);
-          return gzip(contentsBuffer).then((buffer) => ({
-            name: file,
-            size: contentsBuffer.length,
-            gzipSize: buffer.length,
-            showGzipped: contentsBuffer.length > 0,
-          }));
+          return pool.exec('gzipStats', [file]).catch((err) => console.error(err));
```

Comment:
Do we need this `.catch`? What does it provide us? I _think_ it is emitting the error then swallowing the failure, is that intentional?

Previously, an error during this computation would have failed the command (which as far as I can tell is what would happen if this `.catch(err => console.error(err))` was removed). 

Lets try removing it:

```suggestion
          return pool.exec('gzipStats', [file]);
```